src/LaravelRequestDocs.php

```diff
@@ -189,7 +189,12 @@ public function getControllersInfo(array $onlyMethods): Collection
                 $controllerName     = (new ReflectionClass($controllerFullPath))->getShortName();
             }
 
-            $pathParameters = $this->routePath->getPathParameters($route);
+            $pathParameters = [];
+            $pp = $this->routePath->getPathParameters($route);
+            // same format as rules
+            foreach ($pp as $k => $v) {
```

Comment:
format changed to be able to use same UI component

From
```
        "path_parameters": {
            "whatever_id": "string|required|regex:/[0-9]+/"
        },
```


To
```
        "path_parameters": {
            "whatever_id": [
                "string|required|regex:/[0-9]+/"
            ]
        },
```

Reply:
@kitloong 
I ll merge it, can you fix tests upon the format change?
Seems like there are a lot of tests failing on this simple change.

Overall frontend is ready